### DISCUSSION TITLE
Increase the saturations for joint torques of CAN robots

### DISCUSSION THREAD

**Discussion by DanielePucci** [robotology/community#8] [Q&A] [answered]
Does anyone know how to increase the saturations of the joint torques for CAN robots?

**Comment by DanielePucci** [comment, marked as answer]
Yes, 
to do this, you have to change two tags' value in two different  configuration files of the robotInterface.

1) Go on the PC104 (e.g. ssh pc104)

2) Open the configuration xml file under .local/share/yarp/robots/iCubGenova0X/hardware/motorControl associated with the part you want to increase the torque saturations of (e.g. icub_torso.xml). Then change the value of the tag "TorqueMax" into the new saturation value, say x.

3) Open the configuration xml file under .local/share/yarp/robots/iCubGenova0X/hardware/VFT associated with the part you want to increase the torque saturations of (e.g. torso_virtual_strain.xml). Then change the value of the tag "fullScale" into the new saturation value, that is x.

**Comment by pattacini** [comment]
:+1:
